src/core/DropdownMenu/index.tsx

```diff
@@ -67,6 +68,7 @@ interface ExtraDropdownMenuProps extends StyleProps {
   PaperComponent?: typeof StyledPaper | RenderFunctionType;
   children?: JSX.Element | null;
   onClickAway: (event: MouseEvent | TouchEvent) => void;
+  clickAwayListenerProps?: Partial<ClickAwayListenerProps>;
```

Comment:
nit: Can we use capital `C` instead to signify it's a component prop please 🙏 ? Following what MUI does and other props we have! Thanks so much!

```suggestion
  ClickAwayListenerProps?: Partial<ClickAwayListenerProps>;
```

Reply:
Aah, for sure. Thanks for spotting this. 🥳

Reply:
It appears that there is already a ClickAwayListenerProps defined in MUI and is a reserved name. Therefore, we need to come up with a different name for our prop. Would it make sense to rename it to SdsClickAwayListenerProps instead? @tihuan